### PR TITLE
[RPS-23] Add AWS environment foundations

### PR DESCRIPTION
**Pull request by wojtek-oledzki**
To build environmnet foundations run:
`make stack ROLE=aws_foundation`

### CODE REVIEW

**Comment by wojtek-oledzki** [comment]
@ivorc - this is just early preview. I'm still **working on it**. Bastion box stack should be added very shortly.

**Comment by wojtek-oledzki** [comment]
@ivorc this is ready for review

File: ansible/roles/service/bastion/files/bastion.yml [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+              - Action:
+                  - ec2:Describe*
+                  - route53:ListHostedZones
+                Effect: Allow
+                Resource: "*"
+              - Action:
+                  - s3:GetObject
+                  - s3:ListBucket
+                Effect: Allow
+                Resource: !Sub "arn:aws:s3:::config.${Environment}.digital.nhs.uk/bastion/*"
```
Comment (wojtek-oledzki) [inline ansible/roles/service/bastion/files/bastion.yml]: use `ConfigBucket` here

**Review by ivorc — APPROVED** [review]
👍

**Pull request merged**